docs/no_peeking/mappings/biosample.py

```diff
@@ -0,0 +1,94 @@
+"""
+This module maps Kids First biospecimen to Phenopackets Biosample (derived from FHIR Specimen).
+Please visit https://aehrc.github.io/fhir-phenopackets-ig/StructureDefinition-Biosample.html
+    for the detailed structure definition.
+"""
+from kf_lib_data_ingest.common import constants
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+
+
+def biosample_status(x):
+    """
+    http://hl7.org/fhir/R4/valueset-specimen-status.html
+    """
+    if x == constants.COMMON.FALSE:
+        return 'unavailable'
+    elif x == constants.COMMON.TRUE:
+        return 'available'
+    else:
+        raise Exception('Unknown Biosample status')
+
+def biosample_type(x):
+    """
+    http://terminology.hl7.org/ValueSet/v2-0487
```

Comment:
How are these URLs in the comments used?  This URL doesn't resolve but it is taken from http://hl7.org/fhir/R4/v2/0487/index.html as the CodeSystem URL.

Reply:
For all the custom functions defined, the URLs in the docstrings are not for direct use, but for reference. And, when I created this PR, it was before the Phenopackets team has made the recent IG updates and the URLs were valid. 

@fiendish will create a PR to this branch which resolves terminology issues.